### PR TITLE
feat: disabling windows updates by default

### PR DESCRIPTION
**Pull request by marosset**
**Reason for Change**:

Enabling automatic updates on Windows nodes has caused problems for cluster operators in the past and goes against guidance provided by the aks-engine team so it should not be enabled by default in aks-engine

**Issue Fixed**:

**Requirements**:

- [x] uses [conventional commit messages](https://www.conventionalcommits.org/)
  
- [x] includes documentation
- [x] adds unit tests
- [ ] tested upgrade from previous version

**Notes**:

### CODE REVIEW

File: pkg/api/const.go [hunk 1]
```diff
@@ -223,7 +223,7 @@ const (
 	// https://docs.microsoft.com/en-us/azure/azure-subscription-service-limits#load-balancer.
 	DefaultMaximumLoadBalancerRuleCount = 250
 	// DefaultEnableAutomaticUpdates determines the aks-engine provided default for enabling automatic updates
-	DefaultEnableAutomaticUpdates = true
+	DefaultEnableAutomaticUpdates = false
```
Comment (jackfrancis) [inline pkg/api/const.go:226]: For some reason I thought this already defaulted to false, so lgtm

PS this was being unit tested :)

Comment (marosset) [reply pkg/api/const.go:226]: ooo - i'll update

Comment (marosset) [reply pkg/api/const.go:226]: One funny thing is that we disable windows updates in the aks-engine VHDs we produce in the windows registry. I thought this was defaulting to false too.

Comment (jackfrancis) [reply pkg/api/const.go:226]: Ah, that's why I thought we have been doing this for a long time. I still think this change is good for folks who choose not to use the VHD.

**Pull request merged**